executor/sort.go

```diff
@@ -193,9 +193,25 @@ func (e *SortExec) fetchRowChunks(ctx context.Context) error {
 		e.rowChunks.GetDiskTracker().AttachTo(e.diskTracker)
 		e.rowChunks.GetDiskTracker().SetLabel(memory.LabelForRowChunks)
 	}
+	defer func() {
+		if e.rowChunks.NumRow() > 0 {
+			err2 := e.rowChunks.Sort()
```

Comment:
is it necessary to call `e.rowChunks.Sort()` if err is not nil?

Reply:
addressed.